src/apiCalls.js

```diff
@@ -0,0 +1,19 @@
+function fetchSingleMovieData() {
```

Comment:
This needs work, I could not figure out how to get the fetch request to work in an apicalls file

Reply:
Not sure if this would help but I've been using export const for all fetch calls instead of the function and then export default